scripts/cli/mvp_web.py

```
@@ -0,0 +1,456 @@
+
+        # 5) Grade
+        grade_args = _cli_base_args() + [
+            "--mode", "grade",
+            "--chunks-jsonl", str(describe_dir / "chunks.jsonl"),
+            "--retrieval-out-jsonl", str(retrieval_out),
+            "--output-root", str(output_root),
+            "--run-id", run_id,
+            "--grading-provider", provider,
+            "--grading-model", model,
+            "--student-path", student_submission_path.name if student_submission_path else student_upload.name,
+        ]
```

Ruff (0.15.9)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)